scripts/extract_download_logs.py

```diff
@@ -102,11 +162,13 @@ def parse_item(item: dict) -> dict:
     response = cloudwatch_logs_client.get_query_results(queryId=query_id)
 
 rows_dict = cloudwatch_logs_to_rows_dict(response["results"])
```

Comment:
`rows_dict` is a bit of a misleading variable name as it suggests the variable is a dictionary, when we are actually dealing with a list of dictionaries. We should also change the function name `cloudwatch_logs_to_rows_dict`, as well as `rows_dict_to_csv` and the argument name `rows_dict` in that function

Reply:
fixed in: d2f0fba